package.json

```diff
@@ -6,7 +6,7 @@
     "lerna": "^3.15.0"
   },
   "scripts": {
-    "bootstrap": "f () { npm ci && lerna bootstrap --scope deriv-${1:-'*'} && npm run build:travis ;}; f",
+    "bootstrap": "f () { npm ci && lerna bootstrap --scope deriv-${1:-'*'} && npm run build:travis && lerna link ;}; f",
```

Comment:
related to this PR?

Reply:
@reza-fs well its a small change, doesn't require a separate PR.